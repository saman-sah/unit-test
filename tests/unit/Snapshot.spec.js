import { mount } from "@vue/test-utils";
import Snapshot from "@/components/Snapshot.vue"


describe("Snapshot.vue", ()=> {
    it("render corectly", ()=> {
        const wrapper= mount(Snapshot, {
            props: {
                msg: "Different Message"
            }
        })
        expect(wrapper.element).toMatchSnapshot()
    })
})