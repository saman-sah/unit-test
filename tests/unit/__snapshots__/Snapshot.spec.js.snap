// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot.vue render corectly 1`] = `
<h1>
  Different Message
</h1>
`;
